modules/virtualization/microvm/appvm.nix

```diff
@@ -156,6 +156,13 @@ in {
               '';
               type = int;
             };
+            appvmExtraModules = mkOption {
```

Comment:
I think we could simply call this extraModules, and maybe we could also remove the common extraModules which configures settings for all AppVMs

Reply:
I'm not sure why the extraModules were left as the way is it is in line 164, but it is probably remnant of copy-pasting code from other VMs in the past

Reply:
I have no idea what the common extraModules does. It was added/taken into use after my original pull request of this feature. I had to redo and retest this patch because of that change.
I cannot remove the common extraModules as i don't know what it is supposed to do and how to test its possible effects.

Reply:
Could you please change the variable name from `appvmExtraModules`to `extraModules`. I can then write another patch, which removes the common extraModules.

Reply:
Done.